anaconda_project/internal/cli/environment_commands.py

```diff
@@ -226,3 +230,44 @@ def main_update(args):
 def main_unlock(args):
     """Unlock dependency versions and return exit status code."""
     return unlock(args.directory, args.name)
+
+
+def running_in_bootstrap_env(project_dir):
```

Comment:
Minor nitpick, but if this is returning True or False a name `is_running_in_bootstrap_env` is better :-p

Reply:
+1